bin/publish_build

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+echo -e "CURRENT_BRANCH: ${TRAVIS_BRANCH}\n"
+echo -e "PULL_REQUEST: ${TRAVIS_PULL_REQUEST}\n"
+
+# build the distribution files first
+$(npm bin)/grunt build
+
+git config --global user.email "antonio@brandwatch.com"
+git config --global user.name "Antonio D'Ettole"
+
+# This specifies the repository we are going to work with.
+DIST_REPO_SLUG="BrandwatchLtd/selleckt-builds"
+
+# This specifies the user who is associated to the GH_TOKEN
+USER="codazzo"
```

Comment:
This can be changed to anything, e.g. a "builds user" for brandwatch. The user has to match the github token configured as an environment variable in Travis.